chainercv/extensions/__init__.py

```diff
@@ -1 +1,2 @@
 from chainercv.extensions.detection.detection_vis_report import DetectionVisReport  # NOQA
+from chainercv.extensions.detection.detection_voc_evaluator import DetectionVOCEvaluator  # NOQA
```

Comment:
How about `DetectionEvaluatorVOC`? When we add other metrics, we can write `eval_detection_*` and `DetectionEvaluator*`.